state/allcollections.go

```diff
@@ -160,13 +160,6 @@ func allCollections() CollectionSchema {
 		// -----
 
 		// These collections hold information associated with applications.
-		charmsC: {
-			indexes: []mgo.Index{{
-				Key: []string{"model-uuid"},
-			}, {
-				Key: []string{"bundlesha256"},
-			}},
-		},
```

Comment:
That's it for the epic, we're now just fixing bugs!

Well done @jack-w-shaw and @nvinuesa